src/lib/project-saver-hoc.jsx

```diff
@@ -86,6 +94,15 @@ const ProjectSaverHOC = function (WrappedComponent) {
         }
         componentWillUnmount () {
             this.clearAutoSaveTimeout();
+            window.onbeforeunload = undefined; // eslint-disable-line no-undefined
+        }
+        leavePageConfirm (e) {
+            if (this.props.projectChanged) {
+                // both methods of returning a value may be necessary for browser compatibility
+                (e || window.event).returnValue = true;
+                return true;
+            }
+            return false; // do not prompt
```

Comment:
I don't think this should return false, the spec linked in the other comment says that the function should return undefined, so this return can be removed (i.e. return undefined)